Library/Homebrew/cmd/update-report.rb

```diff
@@ -104,6 +105,32 @@ def update_report
       next unless tap.git?
       next if tap.core_tap? && ENV["HOMEBREW_INSTALL_FROM_API"].present? && args.preinstall?
 
+      # HOMEBREW_CORE_MERGE_MAINTAINER can be set by the maintainers
+      # who merge homebrew-core into linuxbrew-core to avoid being migrated
+      # We only migrate HOMEBREW_DEVELOPER users right now
+      # Don't migrate on CI either
+      if ENV["HOMEBREW_DEVELOPER"].present? &&
+         ENV["HOMEBREW_CORE_MERGE_MAINTAINER"].blank? &&
+         ENV["GITHUB_ACTIONS"].blank? &&
+         tap.core_tap? &&
+         tap.remote == "https://github.com/Homebrew/linuxbrew-core"
+        puts_stdout_or_stderr "Migrate from linuxbrew-core to homebrew-core" unless args.quiet?
+
+        LINUXBREW_CORE_MIGRATION_LIST.each do |name|
+          formula = Formula[name]
+          next unless formula.any_version_installed?
+
+          keg = formula.installed_kegs.sort_by(&:version).last
```

Comment:
```suggestion
          keg = formula.installed_kegs.last
```

because it is already sorted by version